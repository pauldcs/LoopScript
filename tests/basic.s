loop (1, 1); 
	move
		(540, 240) # range x 
		(540, 550) # range y
		1.0,  # duration
		1.5;  # wait
	click right;
	loop (3, 3); 
		click right;
		move
			(100, 200)
			(300, 400)
			0.5,
			1.0;
		click left;
		type "Hello world";
		rtext (10, 20); 
	end;
	rstr (5, 10);
end;