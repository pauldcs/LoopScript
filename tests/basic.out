move (540, 240), (540, 550), 1.0, 1.5
click (right)
click (right)
move (100, 200), (300, 400), 0.5, 1.0
click (left)
type 'Hello world'
rtext (10, 20)
click (right)
move (100, 200), (300, 400), 0.5, 1.0
click (left)
type 'Hello world'
rtext (10, 20)
click (right)
move (100, 200), (300, 400), 0.5, 1.0
click (left)
type 'Hello world'
rtext (10, 20)
rstr (5, 10)
